packages/client/hmi-client/src/components/documents/Document.vue

```diff
@@ -1,134 +1,237 @@
+				<div class="scroll-to-section-links">
+					<a>Top</a>
+					<a v-if="!isEmpty(formattedAbstract)">Abstract</a>
+					<a v-if="doc?.knownEntities?.summaries?.sections">Section summaries</a>
+					<a v-if="!isEmpty(figureArtifacts)"
+						>Figures <span class="artifact-amount">({{ figureArtifacts.length }})</span></a
+					>
+					<a v-if="!isEmpty(tableArtifacts)"
+						>Tables <span class="artifact-amount">({{ tableArtifacts.length }})</span></a
+					>
+					<a v-if="!isEmpty(equationArtifacts)"
+						>Equations <span class="artifact-amount">({{ equationArtifacts.length }})</span></a
+					>
+					<a v-if="!isEmpty(githubUrls)"
+						>GitHub URLs <span class="artifact-amount">({{ githubUrls.length }})</span></a
+					>
+					<a v-if="!isEmpty(urlArtifacts)"
+						>Other URLs <span class="artifact-amount">({{ urlArtifacts.length }})</span></a
+					>
+					<a v-if="!isEmpty(otherArtifacts)"
+						>Other extractions <span class="artifact-amount">({{ otherArtifacts.length }})</span></a
+					>
+					<a v-if="!isEmpty(doc.citationList)"
+						>References <span class="artifact-amount">({{ doc.citationList.length }})</span></a
+					>
+					<a v-if="!isEmpty(relatedTerariumArtifacts)"
+						>Associated resources
+						<span class="artifact-amount">({{ relatedTerariumArtifacts.length }})</span></a
+					>
 				</div>
-			</AccordionTab>
-			<AccordionTab v-if="!isEmpty(githubUrls)">
-				<template #header>
-					Github URLs<span class="artifact-amount">({{ githubUrls.length }})</span>
-				</template>
-				<ul>
-					<li class="github-link" v-for="(url, index) in githubUrls" :key="index">
-						<import-code-button v-if="isEditable" :urlString="url" @open-code="openCode" />
-						<a :href="url" rel="noreferrer noopener">{{ url }}</a>
-					</li>
-				</ul>
-			</AccordionTab>
-			<AccordionTab v-if="!isEmpty(urlArtifacts)">
-				<template #header>
-					URLs<span class="artifact-amount">({{ urlArtifacts.length }})</span>
-				</template>
-				<ul>
-					<li v-for="ex in urlArtifacts" :key="ex.url">
-						<b>{{ ex.resourceTitle }}</b>
-						<div>
-							<a :href="ex.url" rel="noreferrer noopener">{{ ex.url }}</a>
-						</div>
-					</li>
-				</ul>
-			</AccordionTab>
-			<AccordionTab v-if="!isEmpty(otherArtifacts)">
-				<template #header>
-					Others<span class="artifact-amount">({{ otherArtifacts.length }})</span>
-				</template>
-				<div v-for="ex in otherArtifacts" :key="ex.askemId" class="extracted-item">
-					<b v-html="highlightSearchTerms(ex.properties.title)" />
-					<span v-html="highlightSearchTerms(ex.properties.caption)" />
-					<span v-html="highlightSearchTerms(ex.properties.abstractText)" />
-					<span v-html="highlightSearchTerms(ex.properties.contentText)" />
+
+				<!-- TODO: Add search on page function (highlight matches and scroll to the next one?)-->
+				<div class="p-input-icon-left">
+					<i class="pi pi-search" />
+					<InputText placeholder="Find in page" class="find-in-page" />
```

Comment:
Remove button or hide them if the functionality doesn't exist yet.
Also, create an issue about creating such task.